src/main/java/aeneas/models/Model.java

```diff
@@ -91,6 +94,11 @@ public void updateStats() {
       if(getMetadata(activeLevel).getStarsEarned() < stars) {
```

Comment:
All this star logic seems odd.
What condition are we describing?
My understanding is that once we win a level (placed all pieces in bullpen, ran out of time, ect...) We calculate the number of stars. If it's > 0, then we unlock the next level.

This conditional trips me up. It checks the current stored number of stars against the current number? Why?